src/tests/functional/plugin/shared/src/behavior/compiled_model/device.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/compiled_model/device.hpp"
+
+namespace ov {
+namespace test {
+namespace behavior {
+
+INSTANTIATE_TEST_SUITE_P(OVCompiledModel, OVCompiledModelCorrectDevice, ::testing::Values("CPU", "GPU", "MULTI:CPU(4),GPU(4)"));
+
+
+TEST_P(OVCompiledModelCorrectDevice, CompileModelWithCorrectDevice) {
+    ov::Core ie = createCoreWithTemplate();
+    OV_ASSERT_NO_THROW(auto compiled_model = ie.compile_model(actualNetwork, deviceName));
```

Comment:
```suggestion
    OV_ASSERT_NO_THROW(std::ignore = ie.compile_model(actualNetwork, deviceName));
```
To ignore return value if required to get but not used.